fixtures/sub-process-no-killable

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
```

Comment:
We don't need to test that `SIGTERM` is received. We explicitly call `kill()` in the tests so we can safely assume child process receive it.

I.e. this fixture is not needed, you could use `fixtures/forever` instead.